lib/ace/mode/d_highlight_rules.js

```diff
@@ -293,7 +293,7 @@ var DHighlightRules = function() {
                 regex: '[a-zA-Z]+'
             }, {
                 token: 'string',
-                regex: '".*"'
+                regex: '"[^"]*"'
```

Comment:
[nit] You could write `'".*?"'` as an alternative

Reply:
Does that work the same in this case? In general I try to avoid backtracking patterns as much as possible, because sometimes innocent looking pattern can freeze the browser https://github.com/ajaxorg/ace/pull/3810/files, but perhaps i was overdefensive in this case.

Reply:
Well, I've never faced any issues with using `.*?` but I think it's ok to be defensive :)